catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -149,6 +180,28 @@ define([
 
                 this.set(newResult);
             }
+            if (this.isUtmUpperLeftDefined()) {
+                var utmParts = this.parseUtmUpperLeft();
+                if (utmParts !== undefined) {
```

Comment:
Looks like the code in this conditional (and the conditional itself) is duplicated for upper left and lower right and could be consolidated into one function